libs/common/src/middleware/front-end-error-handler.get-error-type.test.ts

```diff
@@ -0,0 +1,25 @@
+import createHttpError from 'http-errors';
+import { HttpStatusCode } from 'axios';
+import { getErrorType } from './front-end-error-handler';
+
+describe('front-end-error-handler', () => {
+  beforeEach(() => {});
+
+  describe('getErrorType', () => {
+    it('returns CSRF_TOKEN_ERROR if the error is a CSRF token error', () => {
+      getErrorType(createHttpError(HttpStatusCode.ImATeapot, { code: 'EBADCSRFTOKEN' }));
+    });
+
+    it('returns UNHANDLED_ERROR if the error is an a non CSRF http error', () => {
+      getErrorType(createHttpError(HttpStatusCode.ImATeapot, { code: 'NOT_EBADCSRFTOKEN' }));
+    });
+
+    it('returns UNHANDLED_ERROR if the error is an a non CSRF http error', () => {
```

Comment:
```suggestion
    it('returns UNHANDLED_ERROR when the error is not a CSRF http error', () => {
```

Reply:
I've updated these to the describe blocks I know Zain likes too -- lmk

```
  front-end-error-handler
    getErrorType
      when the error is a CSRF token error
        √ returns CSRF_TOKEN_ERROR (3 ms)
      when the error is an a non CSRF http error
        √ returns UNHANDLED_ERROR
      when the error is an a CSRF non http error
        √ returns UNHANDLED_ERROR (1 ms)
      when the error is an unhandled error
        √ returns UNHANDLED_ERROR (1 ms)
```